lib/auth.ex

```diff
@@ -7,48 +7,74 @@ defmodule FirebaseAdminEx.Auth do
   @doc """
   Get a user's info by UID
   """
-  @spec get_user(String.t()) :: tuple()
-  def get_user(uid), do: get_user(:localId, uid)
+  @spec get_user(String.t(), String.t()) :: tuple()
+  def get_user(uid, client_email \\ nil)
+  def get_user(uid, client_email), do: get_user(:localId, uid, client_email)
```

Comment:
You don't need to have two `def get_user` lines here. You can just do

```elixir
def get_user(uid, client_email \\ nil), do: get_user(:localId, uid, client_emial)
```

Remember also that the `@spec ` should now be
```elixir
@spec get_user(String.t(), String.t() | nil) :: tuple()
```

Same applies to the other functions below